framework/app.mk

```diff
@@ -262,6 +262,7 @@ app_LIBS_other := $(filter-out $(app_LIB),$(app_LIBS))
 app_HEADERS    := $(app_HEADER) $(app_HEADERS)
 app_INCLUDES   += $(app_INCLUDE) $(ADDITIONAL_INCLUDES)
 app_DIRS       += $(APPLICATION_DIR)
+app_LINKS      := $(link_names) $(app_LINKS)
```

Comment:
what does this include?

I know the Makefile isnt very commented but let's comment that one